modules/c++/mt/include/mt/ThreadGroup.h

```diff
@@ -89,13 +85,31 @@ class ThreadGroup
      */
     void joinAll();
 
+    /*!
+     * Checks whether CPU pinning is set (i.e. whether mAffinityInit is NULL)
+     *
+     * \returns true if CPU pinning is set, false otherwise
```

Comment:
Use `\return` rather than `\returns` (for consistency with existing documentation)

Reply:
Done. Might be worth checking documentation consistency; I see `\returns` in a few places in the existing docs.